### PR TITLE
Create CosmosDB.csx

### PR DESCRIPTION
**Pull request by yunjchoi**

### CODE REVIEW

File: data/templates/Detector_CosmosDB.csx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+private static string GetQuery(OperationContext<AzureCosmosDB> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('Aks').database('AKSprod').<YOUR_TABLE_NAME>
```
Comment (khaled-zayed) [inline data/templates/Detector_CosmosDB.csx:7]: I think we should update those to some cosmosdb ones

File: data/templates/Detector_CosmosDB.csx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        | where Timestamp >= startTime and Timestamp <= endTime
+        | <YOUR_QUERY>";
+}
+
+[AzureCosmosDBFilter]
```
Comment (khaled-zayed) [inline data/templates/Detector_CosmosDB.csx:12]: I believe this should be an ArmResourceFilterlike this:
[ArmResourceFilter(provider: "Microsoft.DocumentDB", resourceTypeName: "databaseAccounts")]

**Review by khaled-zayed — CHANGES_REQUESTED** [review]

File: data/templates/Detector_CosmosDB.csx (hunk 1 above)
Comment (yunjchoi) [reply data/templates/Detector_CosmosDB.csx:7]: Thanks for catching this!

File: data/templates/Detector_CosmosDB.csx (hunk 2 above)
Comment (yunjchoi) [reply data/templates/Detector_CosmosDB.csx]: I updated the file with your suggestion.

**Review by khaled-zayed — APPROVED** [review]

**Pull request merged**